File: WebApp/Scripts/app/model/BC_WHPICK.js
Ext.define('WEBAPP.model.BC_WHPICK', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },
        { name: 'PICK_DATE', type: 'date' },
        { name: 'DOCNO', type: 'string' },
        { name: 'SEQ', type: 'string' },
        { name: 'MMCODDE', type: 'string' },
        { name: 'APPQTY', type: 'string' },
        { name: 'BASE_UNIT', type: 'string' },
        { name: 'APLYITEM_NOTE', type: 'string' },
        { name: 'MAT_CLASS', type: 'string' },
        { name: 'MMNAME_C', type: 'string' },
        { name: 'MMNAME_E', type: 'string' },
        { name: 'WEXP_ID', type: 'string' },
        { name: 'STORE_LOC', type: 'string' },
        { name: 'PICK_USERID', type: 'string' },
        { name: 'PICK_SEQ', type: 'string' },
        { name: 'ACT_PICK_USERID', type: 'string' },
        { name: 'ACT_PICK_QTY', type: 'string' },
        { name: 'ACT_PICK_TIME', type: 'date' },
        { name: 'ACT_PICK_NOTE', type: 'string' },
        { name: 'HAS_CONFIRMED', type: 'string' },
        { name: 'BOXNO', type: 'string' },
        { name: 'BARCODE', type: 'string' },
        { name: 'XCATEGORY', type: 'string' },

        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        { name: 'LOT_NO', type: 'string' },
        { name: 'WEXP_ID_DESC', type: 'string' }
    ]
});

File: WebApp/Models/BC/BC_WHPICK.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class BC_WHPICK : JCLib.Mvc.BaseModel
    {
        public string WH_NO { get; set; }
        public string PICK_DATE { get; set; }
        public string DOCNO { get; set; }
        public string SEQ { get; set; }
        public string MMCODE { get; set; }
        public string APPQTY { get; set; }
        public string BASE_UNIT { get; set; }
        public string APLYITEM_NOTE { get; set; }
        public string MAT_CLASS { get; set; }
        public string MMNAME_C { get; set; }
        public string MMNAME_E { get; set; }
        public string WEXP_ID { get; set; }
        public string STORE_LOC { get; set; }
        public string PICK_USERID { get; set; }
        public string PICK_SEQ { get; set; }
        public string ACT_PICK_USERID { get; set; }
        public string ACT_PICK_QTY { get; set; }
        public string ACT_PICK_TIME { get; set; }
        public string ACT_PICK_NOTE { get; set; }
        public string HAS_CONFIRMED { get; set; }
        public string BOXNO { get; set; }
        public string BARCODE { get; set; }
        public string XCATEGORY { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string HAS_SHIPOUT { get; set; }
        // ===============================================
        public string LOT_NO { get; set; }
        public string CALC_TIME { get; set; }
        public string PICK_USERNAME { get; set; }
        public string WEXP_ID_DESC { get; set; }

        public string ITEM_STRING { get; set; }
        public string APPDEPT { get; set; }
        public string INV_QTY { get; set; }
        public string APPLY_NOTE { get; set; }
        public string NEED_PICK_ITEMS { get; set; }
        public string LACK_PICK_ITEMS { get; set; }
        public string VALID_DATE { get; set; }
        public string LOT_NO_F { get; set; }
        public string APPDEPTNAME { get; set; }
        public string SHIPOUTCNT { get; set; }


    }
}